### PR TITLE
Add speaker Meng Weng Wong

### LINKED ISSUE #589: Add speaker Meng Weng Wong
Meng Weng Wong
Co-Founder
Legalese.com

other info please find here: https://eventyay.com/e/b548bcfc/#speakers
please use his photo below, thank you!
1024px-meng_weng_wong

### PR DESCRIPTION
**Pull request by kurianbenoy**
fix #589 issue.

Sorry for not squashing commits

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3423,6 +3427,37 @@ <h1>Speakers</h1>
                             <span>True Global Ventures</span>
                         </div>
                     </div>
+
+                    <div class="col-md-3 col-sm-6 speaker-column">
+                        <div class="speaker">
+                            <div class="image-holder">
+                                <img class="background-image" alt="Meng Weng Wong" src="img/mengwengwong.jpg">
```
Comment (simsausaurabh) [inline index.html]: Please follow the naming convention for images across the project. It should be ```MengWengWong.jpg```

Comment (kurianbenoy) [reply index.html]: ok

**Comment by kurianbenoy** [comment]
Made the requested changes

**Comment by simsausaurabh** [comment]
@kurianbenoy Thanks, for the changes. Sorry I wasn't able to review again. Keep up the nice work, cheers!

**Pull request merged**